controls/ssh_spec.rb

```diff
@@ -40,6 +39,16 @@
     it { should_not be_writable.by('group') }
     it { should_not be_writable.by('other') }
   end
+  case os[:family]
```

Comment:
@techraf there is no need to close this PR. You add a commit on top, or (better) use rebase and squash to include the fixups in your commits. Have a look to the [official git doc](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) and [here](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) is a short example.

Please keep in mind, after changing the git history you will have to force push (`git push -f`): the regural push won't work because of changed commit history and 'git confusion'

Usually its a good practice to address review comments via commiting on top via `git commit --fixup=` and to cleanup the history before merge to master with `git rebase -i master --autosquash`. This allows reviewers to see if their comments were addressed (on the easy way) and provides in the same time clean history in the master branch.